### ISSUE TITLE
Soft wrap setting does not affect soft wrap when new file is opened

### ISSUE THREAD

**Issue by irisli** [atom/atom#8015]
I've been having this problem for a while and I just `rm -rf`'ed Atom.app and `~/.atom` then installed fresh from a new download from the atom website. It still has this problem:

![image](https://cloud.githubusercontent.com/assets/5728307/8814127/097705a0-2fbe-11e5-898f-ca44b30ee84b.png)

In both checked and unchecked positions of this checkbox, new documents I open still have soft wrap on by default. Even after I restart it, it still has this problem of not following the settings by default.

**Comment by irisli** [comment]
I know that a workaround is to set a hotkey to toggle soft wrap, but it seems that this checkbox should affect the default.

**Comment by jonleung** [comment]
This is a **HUGE** problem for me. Any suggestions for how to fix this would be appreciated!

**Comment by bbuck** [comment]
I've been having the same issue, I've just noticed it for any really log lines. It's quite frustrating. Although I should mention that while it happens by default if I turn on Soft Wrap in settings it seems to wrap _more_ lines than by default. I've combed through my packages, reinstalled everything from the ground up. This issue along with the tree view being open every time Atom opens has been really frustrating.
